build.assets/build-package.sh

```diff
@@ -74,9 +74,9 @@ FPM_IMAGE_RPM="public.ecr.aws/gravitational/fpm:centos8-1.15.1-1"
 
 # extra package information for linux
 MAINTAINER="info@goteleport.com"
-LICENSE="Apache-2.0"
+LICENSE="AGPL-3.0-or-later"
```

Comment:
The packages we distribute are governed by `build.assets/LICENSE-community`, NOT by AGPLv3.

We should change this (is there a way to use a custom license in a package?), and we should also be including a copy of `LICENSE-community` in the package itself.

Reply:
Ah ok the task we had wasn't entirely clear on what was necessary. Thanks for the additional context.

We should be able to use a custom license. Afaik this just takes in an arbitrary string. Unfortunately there's not really a way to query for the contents of the license itself. Maybe we could provide a link to the license in the description?

Adding the `LICENSE-community` should be easy enough though. Just need to update `build-archive` to include it in the tarball.

Reply:
I suppose its even a little more complicated than that, as the `-ent` packages are not governed by `LICENSE-community`.